docs/package-sign.md

```diff
@@ -84,213 +84,80 @@ Follow these steps to get the packaging and signing tool and set up the virtual
     (.venv) C:\connector-plugin-sdk\connector-packager>python setup.py install  
     ```
 
-## Use keytool to get a certificate
-
-Getting a certificate is a multi-step process.
-
-### Step 1: Use keytool to generate a key pair 
-
-To generate a key pair, run this command:
-
-```
-keytool -genkeypair -alias alias -keystore keystore 
-```
-
-For example: 
-
-```
-(.venv) D:\connector-plugin-sdk\connector-packager>keytool -genkey -alias test1year -keystore test1yearkeystore.jks -validity 365 
-```
-
-For more information about keytool arguments, see the Java Documentation about [keytool](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on the Oracle website. 
-
- 
-
-### Step 2: Export key to certificate file
-
-Run this command to export the key to a certificate file: 
-
-```
-keytool -export -alias alias -file cert_file -keystore keystore 
-```
-
-For example: 
-
-```
-(.venv) D:\connector-plugin-sdk\connector-packager>keytool -export -alias test1year -file test1year.crt -keystore test1yearkeystore.jks 
-```
- 
-
-### Step 3: Generate a Certificate Signing Request 
-
-A certificate signing request (CSR) is a request for a certificate authority (CA) to create a public certificate for your organization. To generate a CSR, run the following command:
-
-```
-keytool -certreq -alias alias -keystore keystore -file certreq_file
-```
-
-For example: 
-
-```
-(.venv) D:\connector-plugin-sdk\connector-packager>keytool -certreq -alias test1year -keystore test1yearkeystore.jks -file test1year.csr 
-```
- 
-
-### Step 4: Get the CSR signed by the certificate authority 
-
-Send the certificate signing request to the CA you want to create a certificate for you (for example, Verisign, Thawte, or some other CA). The CA will sign the CSR file with their own signature and send that certificate back to you. You can then use this signed certificate to sign the .taco file.
-
-After you receive/fetch the new certificate from the CA, along with any applicable 'chain' or 'intermediate' certificates, run the following command to install the new cert and chain into the keystore:
-
-```
-keytool -importcert 
-```
-
-
-## Package and sign the connector
+## Package the connector
 
 
 ### Run the package module 
```

Comment:
Since this is the only subsection in Package the connector, we can remove this subsection title?